tests/wallet/test_signer_protocol.py

```diff
@@ -36,14 +33,120 @@
     TransactionInfo,
     UnsignedTransaction,
 )
-from chia.wallet.util.clvm_streamable import ClvmSerializationConfig, _ClvmSerializationMode, clvm_serialization_mode
+from chia.wallet.util.clvm_streamable import (
+    byte_deserialize_clvm_streamable,
+    byte_serialize_clvm_streamable,
+    clvm_streamable,
+    json_deserialize_with_clvm_streamable,
+    json_serialize_with_clvm_streamable,
+    program_deserialize_clvm_streamable,
+    program_serialize_clvm_streamable,
+)
 from chia.wallet.util.tx_config import DEFAULT_COIN_SELECTION_CONFIG
 from chia.wallet.wallet import Wallet
 from chia.wallet.wallet_state_manager import WalletStateManager
 from tests.environments.wallet import WalletStateTransition, WalletTestFramework
 
 
-def test_signing_serialization() -> None:
+@clvm_streamable
+@dataclasses.dataclass(frozen=True)
+class Temp(Streamable):
```

Comment:
I dunno, probably most options other than `Temp` would be better.

```suggestion
class BasicClvmStreamable(Streamable):
```

Reply:
Suggestion taken